src/components/menu.js

```diff
@@ -3,6 +3,14 @@ import classnames from "classnames";
 
 import styles from "./menu.scss";
 
+const registerToggleActive = (add, toggleActive) => {
+  const frames = document.getElementsByTagName("iframe");
```

Comment:
Isn't this a bit of an overkill? Considering event bubbling I think adding the event on the parent should be enough.

Reply:
@liorheber 
how?

Reply:
It didn't work in storybook.
my solution did

Reply:
how to get the parent?

Reply:
What did you do?

Reply:
I put an iframe as a child of the document

Reply:
What I mean is how did you try and capture the clicks?
Also, isn't the problem you're trying to solve is when the menu is inside an iframe?

Reply:
Not necessarily. 
I want it to toggle the menu when clicking outside the menu... means when clicking on any other iframe on the screen as well

Reply:
Lets take this live as the conversation includes some Kenshoo specifics.